example/demo_catalog.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: "config.istio.io/v1alpha2"
```

Comment:
If we separate the api group by istio component, maybe we can skip `config`?  Like `broker.istio.io/v1alpha2`